spark/dl/src/test/scala/com/intel/analytics/bigdl/python/api/PythonSpec.scala

```diff
@@ -141,6 +141,7 @@ class PythonSpec extends FlatSpec with Matchers with BeforeAndAfter {
   }
 
   "Double prototype" should "be test" in {
+    TestUtils.cancelOnWindows()
```

Comment:
hadoop will try to call chmod underhood and it will cause a failure. So cancel it on windows